docs/index.md

```diff
@@ -1093,10 +1093,26 @@ Can be specified multiple times.
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
 Specifying `--extension` will _remove_ `.js` as a test file extension; use `--extension js` to re-add it. For example, to load `.mjs` and `.js` test files, you must supply `--extension mjs --extension js`.
 
+### `--watch-files <file|directory|glob>`
+
+> _New in v6.3.0_
```

Comment:
Yes, one might consider this a breaking change. I’d defer to you, the maintainers, to make that decision since I don’t know how mocha has handled these changes in the past.

Reply:
@mochajs/core What do you think? 
It looks `semver-major` for me.

Reply:
So it's a backwards-incompatible change to `watch-extensions` (i.e. )? In which case major.

Reply:
> So it's a backwards-incompatible change to `--watch-extensions`?

The only thing that changed with respect to behavior of `--watch-extensions` is that newly created files are now watched, too. Otherwise the behavior is the same.

Reply:
What if `--watch-extensions` and `--watch-files` contradict each other?
I vote for removing `--watch-extensions` and `semver-major` v7.0.0.

Reply:
:+1: